cleanup: update style guide (#11860)
tldr-pages/tldr: contributing-guides/style-guide.md

```

> [!NOTE]

> > The filename and page title must match the command name exactly. Page title can use any case, whereas the filenames must be lowercase.

Author: ```suggestion
> The filename and page title must match the command name exactly. The page title can be present in any case, whereas the filenames must be lowercase.
```

- Avoid using the page title in the description (e.g. use `A sketching and painting program designed for digital artists` instead of `Krita is a sketching and painting program designed for digital artists`) unless the program name differs from the executable name (e.g. `rg` and Ripgrep).
- Avoid mentioning that the program is used on the command-line (e.g. use `Ripgrep is a recursive line-oriented search tool` instead of `Ripgrep is a recursive line-oriented CLI search tool`).
- Brand and project names can be capitalized in the description whenever applicable (e.g. use `A tool for interacting with a Git repository.` instead of ``A tool for interacting with a `git` repository.``).
- Acronym expansions (i.e. protocols, tools, etc) must not be translated unless there is a recognized native equivalent for them.

> - When documenting keycaps or a keyboard shortcut for a utility it is suggested to wrap them in backticks to make them stand out in description (i.e. "Print the last lines of a given file and keep reading file until `Ctrl + C`"). Alternatively, you can document them as a seperate command and optionally highlight them as placeholders (i.e. `:wq{{Enter}}`).

Author: ```suggestion
- When documenting keycaps or a keyboard shortcut for a utility it is suggested to wrap them in backticks to make them stand out in the description (i.e. "Print the last lines of a given file and keep reading file until `Ctrl + C`"). Alternatively, you can document them as a separate command and optionally highlight them as placeholders (i.e. `:wq{{Enter}}`).
```

> - When documenting keycaps or a keyboard shortcut for a utility it is suggested to wrap them in backticks to make them stand out in the description (i.e. "Print the last lines of a given file and keep reading file until `Ctrl + C`"). Alternatively, you can document them as a separate command and optionally highlight them as placeholders (i.e. `:wq{{Enter}}`).

Author: In the example? Or should I rephrase it?

### French-Specific Rules

- Command and example descriptions on pages in French must use the third person singular present indicative tense (présent de l'indicatif à la troisième personne du singulier). For example, use `Extrait une archive` rather than `Extraire une archive` or `Extrais une archive`.

> - There must be a single blank space between special characters in the descriptions. For example, use `Plus d'informations : https://example.com.` instead of `Plus d'informations: https://example.com.` and use `Crée une archive à partir de fichiers :` instead of `Crée une archive à partir de fichiers:`.

Author: Isn't this common for other languages too (except a few ones like Korean)? i.e. we add space after special characters by default. I don't want to redundantly document it we can leave it to the respective contributor's IG.

> - There must be a single blank space between the special character in the descriptions. For example, use `Plus d'informations : https://example.com.` instead of `Plus d'informations: https://example.com.` and use `Crée une archive à partir de fichiers :` instead of `Crée une archive à partir de fichiers:`.

Author: ```suggestion
- There must be a single blank space between special characters in the descriptions. For example, use `Plus d'informations : https://example.com.` instead of `Plus d'informations: https://example.com.` and use `Crée une archive à partir de fichiers :` instead of `Crée une archive à partir de fichiers:`.
```
